TYPO3.Media/Classes/TYPO3/Media/Command/MediaCommandController.php

```diff
@@ -123,6 +175,31 @@ public function clearThumbnailsCommand()
     }
 
     /**
+     * Generate uninitialized asynchronous thumbnails
+     *
+     * @param integer $limit
+     * @return void
+     */
+    public function generateThumbnailsCommand($limit = null)
```

Comment:
maybe it's possible to combine both commands into one? I have to admit that I didn't completely get the difference between "thumbnails with and without preset" yet, after skimming all the code. But it's also been a lot of hours today already ...